packages/cubejs-docker/.gitignore

```diff
@@ -1 +1,4 @@
-dist
+dist/
```

Comment:
Not critical, but in the future we might want a .dockerignore file for them as it will optimize the docker build process.
In general the Docker build step for Cube.js is horrible, it would be nice in the future to optimize it.